students/ak-foster/Lesson04/kata.py

```diff
@@ -0,0 +1,9 @@
+punctuation = "!\"#$%&'()*+,-./:;<=>?@[\]^_`{|}~"
+
+with open('sherlock_small.txt', 'r') as f:
+    contents = f.read()
+    text = contents.split()
+    map = str.maketrans('', '', punctuation)
+    stripped = [words.translate(map) for words in text]
+    print(stripped)
+    print(text)
```

Comment:
Map is a built-in, so best not to use it as a variable name. Also, check out string.punctuation!